src/components/UserDashboard/charts.js

```diff
@@ -71,15 +79,37 @@ const trackBarClick = (trackingEventCategory, barData) => {
   });
 };
 
-export const StudiesChart = withTheme(({ data, theme }) => {
+export const studiesChart = withTheme(({ data, theme, setSqons, virtualStudy }) => {
   const mergedStudyData = data.map(d => ({
     ...d,
     url: getFileRepoURL(SHORT_NAME_FIELD, d.name),
   }));
 
   const onClick = barData => {
     trackBarClick(studiesChartCategory, barData);
-    window.location.href = barData.data.url;
+    addSqon('study.short_name', barData.data.name);
+    window.location.href = '/explore';
```

Comment:
I know you did not wrote that line, but can you change that to `history.push('/explore')` instead?

`history` comes from `react-router` and can be added to the props using the `withRouter` HOC.

We should do that anytime we encounter a line that sets `window.location.href` directly has react-router can fail to catch it, causing a refresh of the page.